src/lib/plugins/placeBlock.js

```diff
@@ -60,23 +65,6 @@ module.exports.server = (serv, { version }) => {
       }
       return data
     }
-
-    // Register default handlers for item -> block conversion
-    for (const name of Object.keys(mcData.itemsByName)) {
-      const block = mcData.blocksByName[name]
-      if (block) {
-        if (block.states.length > 0) {
-          serv.onItemPlace(name, ({ properties }) => {
```

Comment:
did you check this is used nowhere?

Reply:
Yes. It was used to generate part of the stuff that's now in minecraft-data.

Reply:
https://github.com/search?q=repo%3APrismarineJS%2Fflying-squid+serv.onItemPlace%28&type=code it seems used in many places

Reply:
Oh whoops I completely misread the file name. I thought this was the file I deleted.

The `serv.onItemPlace` method is not deleted and there are no side effects to moving the default handlers to the function as far as I am aware.

https://github.com/PrismarineJS/flying-squid/blob/d11b23d8a606c86202127ccd60bd3f023339a8ed/src/lib/plugins/placeBlock.js#L17-L25